Source/CarthageKit/Git.swift

```diff
@@ -57,6 +57,30 @@ public struct GitURL: Equatable {
 		}
 	}
 
+	/// Amount of time before a git repository is fetched again. Defaults to 1 minute
+	public static var fetchCacheInterval: NSTimeInterval = 60.0
```

Comment:
I don't think I understand why this mutable property belongs to `GitURL`, which represents a URL...